packages/cli/src/commands/inspect.ts

```diff
@@ -91,9 +98,42 @@ export async function inspect(
         .map((o) => `${o[0]}=${o[1]}`)
         .join(' ') || '(none)'
     );
+    packageOwner ? console.log('           Owner:', packageOwner) : console.log('          Source:', localSource || '(none)');
     console.log('     Package URL:', deployUrl);
     console.log('        Misc URL:', deployData.miscUrl);
-    console.log('Package Info URL:', metaUrl);
+    console.log('Package Info URL:', metaUrl || '(none)');
+    console.log('Cannon generator:', deployData.generator);
+    console.log('       timestamp:', deployData.timestamp);
+    console.log('IPFS Availability Score(# of nodes): ', ipfsAvailabilityScore || 'Run IPFS Locally to get this score');
```

Comment:
since this new field is longer than all the other ones... im torn on if we should align all of them so that it doesn't look lik ethe availability store is awkwardly sticking out?

maybe add a line in between, like:

```
console.log()
```